libbindgen/tests/tests.rs

```diff
@@ -0,0 +1,150 @@
+extern crate diff;
+#[macro_use]
+extern crate lazy_static;
+extern crate libbindgen;
+extern crate tempdir;
+
+use std::env;
+use std::fs;
+use std::io::{BufReader, BufRead, Read};
+use std::path::{Path, PathBuf};
+use tempdir::TempDir;
+
+lazy_static! {
+    static ref TEMP_DIR: PathBuf = TempDir::new("libbindgen-tests")
```

Comment:
Ah, I left that in because I think I still have to do something about the uses tests. I'll check and rip it out if no longer required.